### ISSUE TITLE
cnnModel.py

### ISSUE THREAD

**Issue by caijiangyao1991** [zhaoyingjun/TensorFlow-Coding#1]
def dropout_flatten_layer(previous_layer, keep_prop):
            dropout = tf.nn.dropout(x=previous_layer, keep_prob=keep_prop)
            num_features = dropout.get_shape()[1:].num_elements()
            layer = tf.reshape(previous_layer, shape=(-1, num_features))  # Flattening the results.
            return layer

里面是不是写错了，应该是tf.reshape(dropout , shape=(-1, num_features))

**Comment by zhaoyingjun** [comment]
是的
